tests/event_test.py

```diff
@@ -69,6 +69,12 @@ def test_room_avatar_event(self):
         event = RoomAvatarEvent.from_dict(parsed_dict)
         assert isinstance(event, RoomAvatarEvent)
 
+    def test_tag_event(self):
+        parsed_dict = TestClass._load_response(
+            "tests/data/events/tag.json")
+        event = TagEvent.from_dict(parsed_dict)
```

Comment:
Using AccountData.parse_event() here would result in testing this new path as well as the one in parse_event().